website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md

```diff
@@ -152,7 +152,8 @@ modifications, which will be detailed below.
         "s3:ListBucket",
         "s3:PutBucketNotification",
         "s3:GetInventoryConfiguration",
-        "s3:PutInventoryConfiguration"
+        "s3:PutInventoryConfiguration",
+        "s3:ListBucketVersions"
```

Comment:
copy_to_glacier's call to list_object_versions didn't work unless I added this.

Reply:
Make sure to add to the migration notes that users should add that permission to their bucket policy documents.

Reply:
I wasn't sure about that, because theoretically all of our users must have something that works, right? This doesn't seem like it should be a new requirement.

Reply:
I would hope so,but I think I would still call it out in our migration notes to have them verify to head off any potential issues by just saying in migration to validate that the following permissions are required on your ORCA bucket.

Reply:
Added